### PR TITLE
Refine api doc

### PR DESCRIPTION
**Pull request by jerrywgz**
Refine docs below:
- anchor_generator 
- bipartite_match
- box_clip
- box_coder
- box_decoder_and_assign
- clip_by_norm
- collect_fpn_proposals
- distribute_fpn_proposals
- maxout
- prelu
- roi_align

### CODE REVIEW

**Comment by jerrywgz** [comment]
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

File: python/paddle/fluid/layers/detection.py [hunk 1]
```diff
@@ -3066,14 +3102,16 @@ def box_decoder_and_assign(prior_box,
         target_box(${target_box_type}): ${target_box_comment}
         box_score(${box_score_type}): ${box_score_comment}
         box_clip(${box_clip_type}): ${box_clip_comment}
-        name(str|None): The name of this operator
+        name(str, optional): The parameter is used for developers to print
```
Comment (heavengate) [inline python/paddle/fluid/layers/detection.py]: name统一说法修改了，这里英文也修改下

Comment (heavengate) [reply python/paddle/fluid/layers/detection.py]: 其余同

File: python/paddle/fluid/layers/detection.py [hunk 2]
```diff
@@ -3122,8 +3160,9 @@ def collect_fpn_proposals(multi_rois,
                           post_nms_top_n,
                           name=None):
     """
-    Concat multi-level RoIs (Region of Interest) and select N RoIs 
-    with respect to multi_scores. This operation performs the following steps:
+    **This OP only supports LoDTensor as input**. Concat multi-level RoIs
```
Comment (heavengate) [inline python/paddle/fluid/layers/detection.py:3164]: Concat -> concatenate

File: python/paddle/fluid/layers/detection.py (hunk 1 above)
Comment (jerrywgz) [reply python/paddle/fluid/layers/detection.py]: Done

**Review by heavengate — APPROVED** [review]

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**